gslib/tests/test_stet_cp.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google Inc. All Rights Reserved.
```

Comment:
Per go/copyright, this should be `Google LLC`.

Reply:
Done, but now every copyright header in the repo is going to make me uncomfortable